library/go/core/metrics/solomon/spack.go

```diff
@@ -110,11 +110,11 @@ func (s *spackMetric) writeLabel(se *spackEncoder, namesIdx map[string]uint32, v
 		return err
 	}
 
-	err = writeULEB128(&s.labels, uint32(namesIdx[name]))
+	err = writeULEB128(&s.labels, namesIdx[name])
```

Comment:
this verbosity is desireable to show what is an actual type, hiding this is not universally benefitial, a lot of people read this code out of theirs IDE-s.